spec.md

```diff
@@ -88,6 +88,16 @@ interface BlockStatement <: Statement {
 
 A block statement, i.e., a sequence of statements surrounded by braces.
 
+## FunctionBody
+
+```js
+interface FunctionBody <: BlockStatement {
+    type: "BlockStatement";
+    body: [ Directive, ..., Statement ];
+}
+```
+
+The body of a function, which is a block statement that may contain a directive prologue.
```

Comment:
How does inheriting from `ExpressionStatement` to _add_ a property without changing anything else (including `type`) break backwards compatibility?